build-configuration/pom.xml

```
@@ -221,7 +221,7 @@
       <version.spring.security>5.1.1.RELEASE</version.spring.security>
       <version.spring5>5.1.3.RELEASE</version.spring5>
       <version.spring.session>2.1.2.RELEASE</version.spring.session>
-      <version.tomcat85>8.5.31</version.tomcat85>
+      <version.tomcat9>9.0.13</version.tomcat9>
```

```suggestion
      <version.tomcat>9.0.13</version.tomcat>
```
I don't see why we need the version in the string. This should allow for single line changes in the future hopefully :)